extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -403,12 +435,16 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
       const adState = this.tryToPlaceAdAfterPage_(pageId);
 
       if (adState === AD_STATE.PLACED) {
+        this.analyticsEvent_(EVENTS.AD_INSERTED,
```

Comment:
Shall we rename to either PLACED or INSERTED for consistency?

Reply:
done.